packages/server/src/listener.js

```diff
@@ -72,4 +70,30 @@ export default class Listener {
     // Set this.listening to true
     this.listening = true
   }
+
+  async serverErrorHandler(e) {
+    if (e.code === 'EADDRINUSE') {
+      consola.warn(`Address \`${this.host}:${this.port}\` is already in use.`)
```

Comment:
For production, we have to throw a FATAL error because the server is not usable and we silently continue listening on a random port. This makes production debug hard